safely_report/survey/xlsform_functions.py

```diff
@@ -29,3 +33,33 @@ def _selected_at(cls, choice_array: list[str], index: int) -> str:
             return choice_array[index]
         except IndexError:
             return ""
+
+    @classmethod
+    def _str_to_int(cls, value: str):
+        return int(value)
+
+    @classmethod
+    def _pulldata(cls, column, default):
+        """
+        Assumes that all data is pulled from the respondent roster table
```

Comment:
It would be helpful to first describe what this function does; and add this assumption as a note.

Reply:
Done